wicket-examples/src/main/java/org/apache/wicket/examples/repeater/repeater.css

```diff
@@ -72,3 +72,16 @@ col.yellow {
 col.brown {
 	background-color: #CC6633;
 }
+
+.toggleable {
```

Comment:
See my comment about CSS class names below.

Reply:
I used a more specific CSS class rule for toggleable, collapsible, and expandable.